front/app/containers/SiteMap/index.tsx

```diff
@@ -312,39 +312,29 @@ const SiteMap = ({ projects, authUser }: Props) => {
                     );
                   })}
                 </ul>
-
-                <H2 ref={userSpaceSection} tabIndex={-1}>
-                  <FormattedMessage {...messages.userSpaceSection} />
-                </H2>
-                <ul>
-                  {isNilOrError(authUser) ? (
-                    <>
-                      <li>
-                        <Link to="/sign-in">
-                          <FormattedMessage {...messages.signInPage} />
-                        </Link>
-                      </li>
-                      <li>
-                        <Link to="/sign-up">
-                          <FormattedMessage {...messages.signUpPage} />
-                        </Link>
-                      </li>
-                    </>
-                  ) : (
+                <>
+                  {!isNilOrError(authUser) && (
                     <>
-                      <li>
-                        <Link to={`/profile/${authUser.attributes.slug}`}>
-                          <FormattedMessage {...messages.profilePage} />
-                        </Link>
-                      </li>
-                      <li>
-                        <Link to="/profile/edit">
-                          <FormattedMessage {...messages.profileSettings} />
-                        </Link>
-                      </li>
```

Comment:
For the SiteMap, given that Sign In / Sign Out are no longer actual routes in the application, they wouldn't belong in this list. So I've removed them (but left in the profile/settings routes once users are logged in of course).